ecsact/cli/commands/build/recipe/cook.cc

```diff
@@ -189,6 +189,13 @@ static auto handle_source( //
 	auto default_plugins_dir = ecsact::cli::get_default_plugins_dir();
 	auto plugin_paths = std::vector<fs::path>{};
 
+	auto out_dir = fs::path("");
```

Comment:
```suggestion
	auto out_dir = fs::path(options.work_dir);
```

I think the default should be the work dir, ya?